### PR TITLE
Update About text

### PR DESCRIPTION
**Pull request by joshmoore**
In preparation for GBI EoE.

### CODE REVIEW

**Comment by joshmoore** [comment]
I'm getting some weird shading on this PR

image

File: ome2024-ngff-challenge/src/About.svelte [hunk 1]
```diff
@@ -10,57 +10,68 @@
...
+    datasets to enable collection statistics, browsing of thumbnails, and opening
+    the datasets with the OME NGFF Validator, for metadata
+    validation and viewing. Please let us know on image.sc or GitHub if you have any issues
+    or ideas.
   </p>
   <p>
-    Perhaps the most important outcome of the Challenge is the establishment of
-    a federated bioimage data system based on OME-Zarr. To our knowledge, this
-    is the largest federated bioimage data collection ever assembled.
+    Looking back, perhaps the most rewording outcome of the Challenge is that with a remarkably
```
Comment (will-moore) [inline ome2024-ngff-challenge/src/About.svelte:71]: Typo `rewording`

File: ome2024-ngff-challenge/src/About.svelte [hunk 2]
```diff
@@ -10,57 +10,68 @@
...
-    Challenge Viewer [URL?}, which incorporates a CSV with locations of
-    datasets, and connects datasets to the OME NGFF Validator, for metadata
-    validation and viewing.
+    To make the collection as accessible as possible, the
+  <a href="https://ome.github.io/ome2024-ngff-challenge">OME-NGFF Challenge Viewer</a>,
+    was built which uses a CSV with locations of
+    datasets to enable collection statistics, browsing of thumbnails, and opening
+    the datasets with the OME NGFF Validator, for metadata
+    validation and viewing. Please let us know on image.sc or GitHub if you have any issues
+    or ideas.
```
Comment (will-moore) [inline ome2024-ngff-challenge/src/About.svelte:68]: This one sentence is a bit too long. How about this.. (wasn't sure about "collection of statistics"). The sentence in [] is optional?

...was built, which loads the CSV files from contributors and combines all the Zarr data into a single collection. Data can be searched by key-word, filtered and sorted by various metadata and browsed with thumbnails [generated on the fly]. Links allow opening of datasets with the OME NGFF Validator for metadata validation and viewing.

Comment (joshmoore) [reply ome2024-ngff-challenge/src/About.svelte:70]: Thanks integrated with some updates if you want to review.

File: ome2024-ngff-challenge/src/Main.svelte [hunk 3]
```diff
@@ -173,12 +173,12 @@
     <div class="summary">
       <h3 style="text-align:center">
         {#if csvUrl == SAMPLES_HOME}
-          This site makes
           <strong style="font-weight:600">{filesizeformat(totalBytes)}</strong>
-          of public bioimage data available in the Zarr v3 format.
-          The OME-NGFF community converted and now hosts
-          this federated collection of the next version OME-Zarr.
+          of public bioimage data in the Zarr v3 format.
+          converted and now hosted by the federated OME-NGFF community
```
Comment (will-moore) [inline ome2024-ngff-challenge/src/Main.svelte:178]: Upper-case `Converted..`  Maybe remove the "now"? Not sure what that means?  "Converted and hosted by the...

Comment (joshmoore) [reply ome2024-ngff-challenge/src/Main.svelte]: Several new versions pushed. You might look at the history to find several different ones.

**Comment by joshmoore** [comment]
Can you fix the CSS in a separate PR, @will-moore?

**Comment by joshmoore** [comment]
Ok. Merging to have this live. Happy to have more changes though.

**Comment by will-moore** [comment]
Merging? Not merged yet!? 
Css fixes at #76

**Pull request merged**